cmd/copyEnumeratorInit.go

```diff
@@ -129,13 +152,10 @@ func (cca *cookedCopyCmdArgs) initEnumerator(jobPartOrder common.CopyJobPartOrde
 			// Attempt to create the container. If we fail, fail silently.
 			err = cca.createDstContainer(dstContainerName, dst, ctx, existingContainers)
 
-			// check against seenFailedContainers so we don't spam the job log with initialization failed errors
-			if _, ok := seenFailedContainers[dstContainerName]; err != nil && ste.JobsAdmin != nil && !ok {
-				logDstContainerCreateFailureOnce.Do(func() {
-					glcm.Info("Failed to create one or more destination container(s). Your transfers may still succeed if the container already exists.")
-				})
-				ste.JobsAdmin.LogToJobLog(fmt.Sprintf("failed to initialize destination container %s; the transfer will continue (but be wary it may fail): %s", dstContainerName, err))
-				seenFailedContainers[dstContainerName] = true
+			if err != nil {
+				// No need to worry about an error from here.
+				_ = createContainerFailureTransfer(dstContainerName,
+					fmt.Sprintf("failed to initialize destination container %s; the transfer will continue (but be wary it may fail): %s", dstContainerName, err))
```

Comment:
The old on-screen wording, of "You transfers may still success if the container already exists" is nice because it gives the user a clearer picture whether they need to be alarmed.  Can you change the wording here to that please?

Reply:
Also, please use a constant or something instead of repeating the same message string in 3 places.